MaxText/pyconfig.py

```diff
@@ -376,6 +376,27 @@ def validate_megablox_parallelism(raw_keys):
                                using_pipeline_parallelism(raw_keys)):
     raise ValueError("Currently we only support Megablox with data parallelism.")
 
+def update_key(raw_keys, model_keys, key):
+  """Update `key` value in `raw_keys` from the value in `model_keys`. """
+  assert key in model_keys and key in raw_keys
+  if key == 'logical_axis_rules':
```

Comment:
Nit: Can we wrap the code handling logical_axis_rules into its own submethod?

Reply:
Done.